funnel/models/project.py

```diff
@@ -191,6 +192,13 @@ class Project(UuidMixin, BaseScopedNameMixin, Model):
         read={'all'},
         datasets={'primary', 'without_parent', 'related'},
     )
+
+    thumbnail_image = with_roles(
+        sa.orm.mapped_column(LargeBinary, nullable=True),
```

Comment:
Use `sa.LargeBinary` and drop the import. It'll keep the module namespace clean of clutter.